compiler/src/iree/compiler/Codegen/Common/EncodingUtils.cpp

```diff
@@ -64,7 +65,10 @@ MaterializeEncodingConversionTarget::MaterializeEncodingConversionTarget(
   markUnknownOpDynamicallyLegal([](Operation *op) {
     auto typeHasEncoding = [](Type t) -> bool {
       auto tensorType = dyn_cast<RankedTensorType>(t);
-      return tensorType && tensorType.getEncoding();
+      if (!(tensorType && tensorType.getEncoding()))
+        return false;
+      // Allow iree_encoding::packed_storage to pass through.
+      return !IREE::Encoding::hasPackedStorageAttr(tensorType);
```

Comment:
I think it is better to check if the encoding is `EncodingAttr`. This materialization pass/utils was designed for data-tiling, so we should scope it to data-tiling encoding attribute. Then it prevents additional changes when we introduce new encodings (e.g., sparse/whatever/etc.).

(At some point, we probably want to rename it as DataTiledEncodingAttr or something else.)